### PR TITLE
 The cars solution

### PR DESCRIPTION
**Pull request by LianaSimonian**

### CODE REVIEW

File: untitled1/src/com/aca/homework/week2/cars/Car.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+public class Car {
+    int year;
+    String brand;
+
+    Car() {
+
+    }
+
+    Car(int y, String s) {
```
Comment (gevorgyanarman) [inline untitled1/src/com/aca/homework/week2/cars/Car.java]: please do not use letters, write readable names instead

File: untitled1/src/com/aca/homework/week2/cars/Cars.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package com.aca.homework.week2.cars;
+
+public class Cars {
+
+    //global case
+    public static Car[] carsManufacturedAfter2010(Car[] cars) {
+        //  Car[] tempCars = new Car[cars.length];
```
Comment (gevorgyanarman) [inline untitled1/src/com/aca/homework/week2/cars/Cars.java]: do not write comments

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]
make changes and merge to devel

**Comment by LianaSimonian** [comment]
Ok

**Pull request merged**